Missing RBAC rule for E2E TestCustomConfiguration

Since #1400 is merged TestCustomConfiguration fails because of a lack of rbac permissions:

> configmaps is forbidden: User "system:serviceaccount:e2e:e2e" cannot create resource "configmaps" in API group "" in the namespace "e2e"

Full log:
```shell
[2019-07-29T09:07:05.020Z] --- FAIL: TestCustomConfiguration (0.15s)
[2019-07-29T09:07:05.020Z]     --- FAIL: TestCustomConfiguration/Create_dictionary_config_map (0.05s)
[2019-07-29T09:07:05.020Z]         require.go:794: 
[2019-07-29T09:07:05.020Z]             	Error Trace:	podtemplate_test.go:73
[2019-07-29T09:07:05.020Z]             	Error:      	Received unexpected error:
[2019-07-29T09:07:05.020Z]             	            	configmaps is forbidden: User "system:serviceaccount:e2e:e2e" cannot create resource "configmaps" in API group "" in the namespace "e2e"
[2019-07-29T09:07:05.020Z]             	Test:       	TestCustomConfiguration/Create_dictionary_config_map
```

--- a/operators/config/e2e/rbac.yaml
+++ b/operators/config/e2e/rbac.yaml
@@ -79,6 +79,7 @@ rules:
       - endpoints
       - secrets
       - persistentvolumeclaims
+      - configmaps
     verbs:
       - get
       - list

